src/ledger/test/LedgerTestUtils.cpp

```diff
@@ -186,6 +190,21 @@ makeValid(DataEntry& d)
     replaceControlCharacters(d.dataName, 1);
 }
 
+void
+makeValid(ClaimableBalanceEntry& c)
+{
+    c.amount = std::abs(c.amount);
+    clampLow<int64>(1, c.amount);
+
+    c.reserve = std::abs(c.reserve);
+    clampLow<int64>(1, c.reserve);
+
+    c.asset.type(ASSET_TYPE_CREDIT_ALPHANUM4);
+    strToAssetCode(c.asset.alphaNum4().assetCode, "CAD");
+
+    c.createdBy = SecretKey::random().getPublicKey();
```

Comment:
The test works if I remove it. I included this so the `createdBy` key would be valid, but I guess this doesn't make a difference because we don't check for account validity in the claimable balance entry.